service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/vm/CreateAzureVmStep.java

```diff
@@ -130,24 +119,36 @@ public StepResult doStep(FlightContext context) throws InterruptedException, Ret
                       .getWorkingMap()
                       .get(AzureVmHelper.WORKING_MAP_NETWORK_INTERFACE_KEY, String.class));
 
+      // TODO:
```

Comment:
This makes sense -- wondering if we also want to put a note in the OpenAPI spec and mark the region param `deprecated`? https://github.com/DataBiosphere/terra-workspace-manager/blob/main/openapi/src/parts/controlled_azure_vm.yaml#L113